dom-weather-app/public/app.js

```diff
@@ -0,0 +1,91 @@
+const cityInput = document.querySelector('#city-input');
+const temperature = document.querySelector('.temperature');
+const dayOrNight = document.querySelector('.time');
+const conditions = document.querySelector('.conditions');
+const icon = document.querySelector('.weather-icon');
+const errorMessage = document.querySelector('#error');
+
+const apiKey = 'qGLMGbgG2tHUnaRGFoJIvHMul7t55Esr';
+
+
+let timeout = null;
+
+cityInput.addEventListener('keyup', () => {
+    clearTimeout(timeout);
+
+    timeout = setTimeout(function () {
+        getCity(cityInput.value);
+    }, 2000);
+})
+
+const getCity = async (inputValue) => {
+
+    const base = 'http://dataservice.accuweather.com/locations/v1/cities/search';
+    const query = `?apikey=${apiKey}&q=${inputValue}`;
+
+    const response = await fetch(base + query)
+    .catch()
```

Comment:
jak coś nie zadziała to bedzie bardzo cicho, dla własnego zdrowia przynajmniej console.error w tym catchu fajnie miec, chyba ze nie dbasz o to czy przejdzie, ale raczej to ważne

Reply:
jasna sprawa - poprawione